### DISCUSSION TITLE
Flyout page menu item navigation

### DISCUSSION THREAD

**Discussion by vardansargsyan92** [PrismLibrary/Prism#2434] [Xamarin.Forms]
HI folks.

I have updated the Prism library  and I have changed master details page to flyout page. The problem is that I am not able to use menu item navigation inside the the flyout page's view model. I am trying to use `await navigationService.NavigateAsync($"NavigationPage/{menuItem.Page}");` but it does not work. So the question is how can I use menu item navigation inside the view model?.

I also noticed that the PageNavigationService does not handle a case for flyout page inside ProcessNavigation method

**Comment by dansiegel** [comment]
FlyoutPage is only supported in Prism 8.1

**Reply by vardansargsyan92** [reply]
HI @dansiegel . Thanks for response.  Are you planning to release  the 8.1 version in the near future?

**Reply by vardansargsyan92** [reply]
I have created a new issue related to flyout page. Should I close it?

https://github.com/PrismLibrary/Prism/issues/2435

**Reply by vardansargsyan92** [reply]
The reason for replacing master details page was this [enhancement].(https://github.com/PrismLibrary/Prism/issues/2239) You have closed it and that is fine because a new version of Prism covers that part.

**Reply by dansiegel** [reply]
We don't like making promises as it boils down to when we feel it's ready and we do this in our spare time. 8.1 is currently available on Sponsor Connect which you can access along with our private Discord server as a GitHub Sponsor.
